src/main/kotlin/uk/gov/justice/digital/hmpps/supportadditionalneedsapi/service/PlanCreationScheduleService.kt

```diff
@@ -178,8 +219,8 @@ class PlanCreationScheduleService(
   }
 
   fun getDeadlineDate(): LocalDate {
-    val todayPlusTen = LocalDate.now().plus(10, ChronoUnit.DAYS)
-    val pesPlusTen = pesContractDate.plus(10, ChronoUnit.DAYS)
+    val todayPlusTen = LocalDate.now().plus(5, ChronoUnit.DAYS)
```

Comment:
You want to use the new WorkingDayService here 😁 👍 
(maybe one for another day 😁 )

Reply:
(and the variable names are now wrong / confusing? 🤷‍♂️ )

Reply:
ha! plusFive <facepalm>

Reply:
updated